### DISCUSSION TITLE
why can't work on smc files?

### DISCUSSION THREAD

**Discussion by sandwitcherz** [ajohns6/Zelda-3-Launcher#7] [Q&A] [answered]
Hello, i would like to know why the smc files ( EUR ) could'nt work here? only sfc ( USA ) ?

thank you for this project ! i'ts a back to the past ^^

**Comment by RadzPrower** [comment, marked as answer]
Those aspects are out of my control. The launcher simply creates a simpler interface to the original project (snesrev's project).

**Reply by Formedras** [reply]
You could potentially include a library to convert headered ROMs (*.smc) to unheadered (*.sfc), but that doesn't really solve the OP's other part of the problem, which is that the project only supports the American 1.0 release.

**Reply by RadzPrower** [reply]
Any issues with the game itself and compatibility need to be directed to snesrev's project.

**Reply by Formedras** [reply]
Right, sorry. I only meant to speak about what little could **potentially** be within your project's scope, which is very very little regarding this issue.
